modules/ipi-install-checking-ntp-sync.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+// ipi-install-installation-workflow.adoc
+
+:_content-type: PROCEDURE
+[id="checking-ntp-sync_{context}"]
+= Optional: Checking NTP server synchronization
+
+The {product-title} installation program installs the `chrony` Network Time Protocol (NTP) service on the cluster nodes. To complete installation, each node must have access to an NTP time server. You can verify NTP server synchronization by using the `chrony` service.
+
+For disconnected clusters, see the _Additional resources_ section.
```

Comment:
I'd recommend specifying which link in particular you're talking about. Also, if you mean for them to abandon this procedure and use that one instead, I'd make that clearer